tenseal/binding.cpp

```diff
@@ -41,46 +37,81 @@ PYBIND11_MODULE(_tenseal_cpp, m) {
         encryption_parameters : parameters to use to create the SEALContext.)",
         py::arg("encryption_parameters"));
 
-    py::class_<BFVVector>(m, "BFVVector")
-        .def(py::init<shared_ptr<TenSEALContext> &, vector<int64_t>>())
-        .def(py::init<shared_ptr<TenSEALContext> &, const std::string &>())
-        .def("size", &BFVVector::size)
+    py::class_<EncryptedVector<int64_t>,
+               std::shared_ptr<EncryptedVector<int64_t>>>(
+        m, "EncryptedVectorInt64", py::module_local());
+    py::class_<EncryptedVector<double>,
+               std::shared_ptr<EncryptedVector<double>>>(
+        m, "EncryptedVectorDouble", py::module_local());
```

Comment:
pybind requires them, because the operations return EncryptedVector instances.

On a second thought, I could change the operations to always return an instance from the current class